.github/workflows/submitBeta.yml

```diff
@@ -1,6 +1,6 @@
 name: Beta Deployment
 env:
-  INDEXER_URL: ${{ secrets.INDEXER_BETA_URL }}
+  INDEXER_URL: ${{ secrets.INDEXER_URL }}
```

Comment:
Use the usual BE url since we still need it for all routes that are not for Discover

Reply:
Cant we also add `INDEXER_BETA_URL: $${{ secrets.INDEXER_BETA_URL }}` apart from the indexer url?